CMakeLists.txt

```diff
@@ -14,10 +14,8 @@
 
 cmake_minimum_required(VERSION 3.17.2)
 
-if (POLICY CMP00091)
-  # Enable MSVC Runtime Library Property
-  cmake_policy(SET CMP0091 NEW)
-endif()
+# Enable MSVC Runtime Library Property
+cmake_policy(SET CMP0091 NEW)
```

Comment:
```suggestion
```

Reply:
Yep, setting cmake_minimum_required() to a version at 3.15 or above makes setting the policy manually redundant. Removing since its no longer necessary.